packages/frontend/src/content/pages/MetaImages/MetaImage.tsx

```diff
@@ -23,7 +23,7 @@ export function MetaImage(props: MetaImageProps) {
           tvl={props.tvl}
           sevenDayChange={props.sevenDayChange}
         />
-        <Chart endpoint={props.apiEndpoint} days={30} />
+        <Chart endpoint={props.apiEndpoint} days={7} />
```

Comment:
Why do we want this change? Why not keep 30 days?

Reply:
Refactor mistake, thanks.